src/worker/lambda/packages/packagePuller.go

```diff
@@ -175,6 +179,29 @@ func (pp *PackagePuller) sandboxInstall(p *Package) (err error) {
 			return err
 		}
 	}
+	//gets the size of the current package
```

Comment:
let's have a config option so we don't compute/log size unless doing so is enabled